Updates for mc RELEASE.2023-09-20T15-22-31Z (#1015)
minio/docs: source/reference/minio-mc-admin/mc-admin-heal.rst

   If the ``TARGET`` bucket or bucket prefix has an active healing scan,
   the command returns the status of that scan.

++++++++++++++++++++

.. dropdown:: Deprecated Arguments

   The following command flags have been deprecated and should only be used under guidance from MinIO Engineers in association with a SUBNET ticket.

   - ``--scan`` 
     
     The type of scan to perform. Specify one of the following supported scan modes:

       - ``normal`` (default)
       - ``deep``

   - ``--recursive, r`` 
     
     Recursively scans for objects in the specified bucket or bucket prefix.

   - ``--dry-run`` 
     
     Inspects the :mc-cmd:`~mc admin heal TARGET` bucket or bucket prefix, but does *not* perform any object healing.

   - ``--force-start, f`` 
     
     Force starts the healing process.

   - ``--force-stop, s`` 
     
     Force stops the healing sequence.

   - ``--remove`` 
     
     Removes dangling objects and data directories in the healing process not referenced by the metadata on a per-drive basis.

> Colors

Reviewer: ```suggestion
Healing Output Color Key
```

Author: Meaningful headers. What a searchable suggestion. 😬

------

> Originally, the healing mechanism output a table that used colors to attempt to differentiate the status of objects in healing.

Reviewer: ```suggestion
Originally, the healing mechanism output a table that used a Green-Yellow-Red-Gray color key to attempt to differentiate the status of objects in healing.
```

> These colors lack much useful detail and have been deprecated in favor of :ref:`healing metrics available at the cluster level <minio-metrics-and-alerts-available-metrics>`.

Reviewer: :thinking: I think we're OK in simply stating that the the color-based output has been deprecated.

```suggestion
These have been deprecated in favor of more detailed :ref:`healing metrics available at the cluster level <minio-metrics-and-alerts-available-metrics>`.
```

Author: Yeah, I wondered about that even as I wrote it.
Taking your suggestion as is for now.

> Originally, the colors conveyed the following intent:

Author: I'm really glad I could contribute the two `the` words to this much better sentence. 😆
